File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassWithDeeplyNestedComplexField/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassWithDeeplyNestedComplexField.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Cars = "cars";
  }

  public static class CAR {
    public static final String TYPE_NAME = "Car";

    public static final String Make = "make";

    public static final String Model = "model";

    public static final String Engine = "engine";
  }

  public static class ENGINE {
    public static final String TYPE_NAME = "Engine";

    public static final String Type = "type";

    public static final String Bhp = "bhp";

    public static final String Size = "size";

    public static final String Performance = "performance";
  }

  public static class PERFORMANCE {
    public static final String TYPE_NAME = "Performance";

    public static final String ZeroToSixty = "zeroToSixty";

    public static final String QuarterMile = "quarterMile";
  }
}

File: graphql-dgs-codegen-core/src/test/resources/kotlin2/dataClassWithDeeplyNestedComplexField/expected/DgsConstants.kt
package kotlin2.dataClassWithDeeplyNestedComplexField.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Cars: String = "cars"
  }

  public object CAR {
    public const val TYPE_NAME: String = "Car"

    public const val Make: String = "make"

    public const val Model: String = "model"

    public const val Engine: String = "engine"
  }

  public object ENGINE {
    public const val TYPE_NAME: String = "Engine"

    public const val Type: String = "type"

    public const val Bhp: String = "bhp"

    public const val Size: String = "size"

    public const val Performance: String = "performance"
  }

  public object PERFORMANCE {
    public const val TYPE_NAME: String = "Performance"

    public const val ZeroToSixty: String = "zeroToSixty"

    public const val QuarterMile: String = "quarterMile"
  }
}
